designs/administrator/olm/operand-details-grouping/index.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Improve grouping and editing on Operand Details view
+
+When creating an operand, fairly complex nesting and hierarchy of fields is used to set values on the operand. Once created however, the Details tab for that operand does not show this hierarchy and all fields are shown at the same flat level, with somewhat of a mix of spec and status descriptors, with some hierarchical section headers missing as well. This can make this view challenging to interpret.
+
+This design describes surfacing the common spec descriptors first, the status descriptors next, then the rest of the spec descriptors, with a divider separating them. Groups of spec descriptors will now have their grouping reflected using a box for the first level, with level indicator labels conveying any further hierarchy. An option to edit these groups using their existing form view will also be available.
+
+## Separating Spec and Status descriptors
+
+![Dividing groups](img/8-1-divider.png)
+- A new divider will be added between the status and spec descriptor sections to visually separate them.
+
+## Grouping Spec descriptors
+
+![Conveying groups](img/9-1-group.png)
+- The top level of spec descriptors' hierarchy will now be shown grouped in a box.
+- Each field has a level indicator to exactly convey its hierarchy, which would be important for fields beyond the first depth level.
+
+![Conveying groups popover](img/9-2-groupPop.png)
```

Comment:
Would it make sense to use the same treatment in the popover and next to the field to represent the hierarchy?

Reply:
It looks like that is actually how it got implemented, so... I agree!

Reply:
We are going to leave this path off the details page for now, but if we do decide to readd it, I agree and we'd try and use this style.